typescript/web/src/utils/tests/data.fixtures.ts

```diff
@@ -1,17 +1,29 @@
+import { SetRequired } from "type-fest";
 import {
   Dataset,
   LabelClass,
   LabelsAggregates,
   Workspace,
+  Image,
+  Label,
 } from "@labelflow/graphql-types";
+import { LabelType } from "../../graphql-types/globalTypes";
 
 export type WorkspaceData = Pick<Workspace, "slug">;
 
 export type DatasetData = Pick<Dataset, "id" | "name" | "slug"> & {
   labelClasses: Omit<LabelClassData, "dataset">[];
+  images: Omit<ImageData, "dataset">[];
   workspace: WorkspaceData;
 };
 
+export type ImageData = SetRequired<
+  Pick<Image, "id" | "name" | "url" | "thumbnail200Url">,
+  "thumbnail200Url"
+> & {
+  dataset: DatasetData;
+};
+
```

Comment:
And here is where trying to base our fixtures on a `Pick` of global data types instead of either the complete types or the generated apollo client types is failing IMO. You have to work-around cross-references and omit or redeclare properties.